src/components/Explore/TokenRow.tsx

```diff
@@ -87,22 +95,15 @@ const HeaderRowWrapper = styled(TokenRowWrapper)`
   border-color: ${({ theme }) => theme.bg3};
   border-radius: 8px 8px 0px 0px;
 
-  @media only screen and (max-width: 390px) {
-    display: none;
-  }
-`
-const FavoriteCell = styled(Cell)`
-  min-width: 40px;
-  color: ${({ theme }) => theme.text2};
-  @media only screen and (max-width: 640px) {
+  @media only screen and (max-width: 410px) {
     display: none;
   }
 `
 const ListNumberCell = styled(Cell)`
   color: ${({ theme }) => theme.text2};
   min-width: 32px;
 
-  @media only screen and (max-width: 390px) {
+  @media only screen and (max-width: 410px) {
```

Comment:
make the breakpoints variables at the top of the file

Reply:
i think for now you can just do..

`LARGE_MEDIA_BREAKPOINT`
`MOBILE_MEDIA_BREAKPOINT`

etc. it doesn't makes sense as an enum to me